.gitlab/llnl-ci-conda.yml

```diff
@@ -17,13 +16,24 @@ stages:
     SITE: "quartz"
     CONDA_ENV_NAME: "quartz-env"
     CONDA_ARCH: "x86_64"
+    CONDA_REQUIREMENTS: .gitlab/conda-requirements.yml
+    PYTHON_MODULE: "python/3.9.12"
 
 .on_ruby:
   variables:
     SITE: "ruby"
     CONDA_ENV_NAME: "ruby-env"
     CONDA_ARCH: "x86_64"
+    CONDA_REQUIREMENTS: .gitlab/conda-requirements.yml
+    PYTHON_MODULE: "python/3.9.12"
```

Comment:
for conda it doesn't matter, since it will install python within the environment based on python version from requirements, thus you can remove `PYTHON_MODULE`

Reply:
Ah, I reverted the commit